superset/docker-compose.yml

```diff
@@ -19,8 +19,6 @@ services:
     volumes:
       - ${PUDL_OUTPUT}/pudl.duckdb:/app/pudl.duckdb
       - ./roles:/app/roles
-    depends_on:
-      - postgres
```

Comment:
Docker throws an error when you run the `prod` profile because the `postgres` services isn't started.